kornia/core/mixin/image_module.py

```diff
@@ -207,12 +223,18 @@ def save(self, name: Optional[str] = None, n_row: Optional[int] = None) -> None:
             name: Directory to save the images.
             n_row: Number of images displayed in each row of the grid.
         """
+        try:
+            from kornia.io import write_image  # pylint: disable=C0415
+            from kornia.utils.image import make_grid  # pylint: disable=C0415
+        except ImportError as exc:
+            raise ImportError("Kornia not installed. Please install Kornia to use this feature.") from exc
```

Comment:
I do not understand this part. Any other solutions, LazyLoading this can also be wierd.

Reply:
why having this anyway? Lower version of Kornia will never have those mixin modules.

Reply:
we can remove the try/catch